dist/elastio-snap-dkms-conf

```diff
@@ -4,6 +4,8 @@ PACKAGE_VERSION="@MODULE_VERSION@"
 # Items below here should not have to change with each driver version
 PACKAGE_NAME="elastio-snap"
 PRE_BUILD="genconfig.sh ${kernelver}"
+# This must be used in POST_INSTALL instead of in POST_BUILD, because DKMS seems to override the actual signature
+POST_INSTALL="sign-module.sh ${kernelver}  /usr/lib/modules/${kernelver}/updates/dkms/${PACKAGE_NAME}.ko"
```

Comment:
/lib/modules instead of /usr/lib/modules

Reply:
Both paths are valid. At least DKMS seems to use both of them. I'll try to update the script anyway, to see what happens:
![image](https://github.com/Axcient/elastio-snap/assets/149574279/165739bf-ffff-48e6-a8a6-38fadcfd21f4)